test/FunctionalTests/Microsoft.OData.Edm.Tests/ExtensionMethods/TargetHelperTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="TargetHelperTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.ExtensionMethods
+{
+    public class TargetHelperTests
+    {
+        private static TargetPathSampleModel sampleModel;
+        private static IEdmModel edmModel;
+        public TargetHelperTests()

```

Comment:
>public TargetHelperTests() [](http://example.com/codeflow?start=8&length=26)

you can use static constructor

static TargetHelperTests(){
   sampleModel = new TargetPathSampleModel();
   edmModel = sampleModel.Model;
}